src/mmw/js/src/modeling/models.js

```diff
@@ -34,10 +34,15 @@ var ProjectModel = Backbone.Model.extend({
         area_of_interest: null,    // GeoJSON
         model_package: '',         // Package name
         taskModel: null,           // TaskModel
-        scenarios: null            // ScenariosCollection
+        scenarios: null,            // ScenariosCollection
+        uid: null
     },
 
     initialize: function() {
+        var scenarios = this.get('scenarios');
+        if (scenarios === null || typeof scenarios === 'string') {
```

Comment:
I think my fix had to do with modifications not scenarios. Not sure what this is about.